doc/source/cd/background.md

```diff
@@ -0,0 +1,869 @@
+$P(\mathbf{z})=P_{ref}(\mathbf{z})$. If the $p$-value
+obtained is below a given threshold, the null is rejected and the
+alternative hypothesis $P(\mathbf{z}) \ne P_{ref}(\mathbf{z})$ is
+accepted, suggesting drift is occurring.
+
+Since $\mathbf{z}$ is often high-dimensional (even a 200 x 200
+greyscale image has 40k dimensions!), performing hypothesis testing in
+the full-space is often either computationally intractable, or
+unsuitable for the chosen statistical test. Instead, the pipeline below
+is often used, with dimension reduction as a pre-processing step.
+
+:::{figure} images/drift_pipeline.png
+:align: center
+:alt: Drift detection pipeline
+
+*Figure inspired by Figure 1 in Failing Loudly: An Empirical Study
+of Methods for Detecting Dataset Shift.*
+:::
+
+### Hypothesis testing
+
+Hypothesis testing involves first choosing a *test statistic*
+$S(\mathbf{z})$, which is expected to be small if the null
+hypothesis $H_0$ is true, and large if the alternative $H_a$
+is true. For observed data $\mathbf{z}$, $S(\mathbf{z})$ is
+computed, followed by a $p$-value
+$\hat{p} = P(\text{such an extreme } S(\mathbf{z}) | H_0)$. In
+other words, $\hat{p}$ represents the probability of such an
+extreme value of $S(\mathbf{z})$ occurring given that $H_0$
+is true. When $\hat{p}\le \alpha$, results are said to be 
+*statistically significant*, and the null 
+$P(\mathbf{z})=P_{ref}(\mathbf{z})$ is rejected. Conveniently, the 
+threshold $\alpha$ represents the desired false positive rate. 
+
+The *test statistics* available in 
+[Alibi Detect](https://github.com/SeldonIO/alibi-detect) can be
+broadly split into two categories; univariate and multivariate tests:
+
+- Univariate:
+
+  - [Chi-Squared](methods/chisquaredrift.ipynb) (for categorical
+    data)
+  - [Kolmogorov-Smirnov](methods/ksdrift.ipynb)
+
+- Multivariate:
+
+  - [Least-Squares Density Difference
+    (LSDD)](methods/lsdddrift.ipynb)
+  - [Maximum Mean Discrepancy (MMD)](methods/mmddrift.ipynb)
+
+When applied to multidimensional data with dimension $d$, the
+univariate tests are applied in a feature-wise manner. The obtained
+$p$-values for each feature are aggregated either via the
+[Bonferroni](https://mathworld.wolfram.com/BonferroniCorrection.html)
+or the [False Discovery
+Rate](http://www.math.tau.ac.il/~ybenja/MyPapers/benjamini_hochberg1995.pdf)
+(FDR) correction. The Bonferroni correction is more conservative and
+controls for the probability of at least one false positive. The FDR
+correction on the other hand allows for an expected fraction of false
+positives to occur. The univariate tests assume each test (i.e. each 
+feature dimension in this case) are independent. Usually this is not 
+the case, resulting in false positive rates (FPR's) up to $d$-times 
+lower than desired, which becomes especially problematic when $d$ is 
+large. On the other hand, the multivariate tests allow us to keep our 
+false positive rates known, at the cost of greater complexity.
```

Comment:
As @jklaise said, a final comment additionally noting that the univariate tests only consider marginal distributions and could miss changes to joint relationships would be useful.